test/utils/CMakeLists.txt

```diff
@@ -12,23 +12,26 @@ set(fyppFiles testutils.fypp)
 
 foreach(infileName IN LISTS fyppFiles)
 
-    string(REGEX REPLACE ".fypp\$" ".f90" outfileName "${infileName}")
+  string(REGEX REPLACE ".fypp\$" ".f90" outfileName "${infileName}")
 
-    set(outfile "${CMAKE_CURRENT_BINARY_DIR}/${outfileName}")
-    set(infile "${CMAKE_CURRENT_SOURCE_DIR}/${infileName}")
+  set(outfile "${CMAKE_CURRENT_BINARY_DIR}/${outfileName}")
+  set(infile "${CMAKE_CURRENT_SOURCE_DIR}/${infileName}")
 
-    add_custom_command(
-        OUTPUT "${outfile}"
-        COMMAND "${FYPP_PROGRAM}" "${infile}" "${outfile}"
-        MAIN_DEPENDENCY "${infile}"
-        VERBATIM)
+  add_custom_command(
+    OUTPUT "${outfile}"
+    COMMAND "${FYPP_PROGRAM}" "${infile}" "${outfile}"
+    MAIN_DEPENDENCY "${infile}"
+    VERBATIM)
 
-    get_filename_component(modName ${outfileName} NAME_WE)
+  get_filename_component(modName ${outfileName} NAME_WE)
 
-    add_library(${modName} ${outfileName})
-    target_compile_options(${modName} PRIVATE -Wall -Wextra -Wpedantic)
-    target_link_libraries(${modName} PRIVATE MPI::MPI_Fortran)
-    target_compile_definitions(${modName} PRIVATE $<$<BOOL:${DLAF_FORTRAN_WITH_MPI_F08}>:DLAF_FORTRAN_WITH_MPI_F08>)
-    set_target_properties(${modName} PROPERTIES Fortran_PREPROCESS "ON") # Use preprocessor on .f90 files
+  add_library(${modName} ${outfileName})
+  target_compile_options(${modName} PRIVATE -Wall -Wextra -Wpedantic)
+  target_link_libraries(${modName} PRIVATE MPI::MPI_Fortran)
+  target_compile_definitions(
+    ${modName}
+    PRIVATE $<$<BOOL:${DLAF_FORTRAN_WITH_MPI_F08}>:DLAF_FORTRAN_WITH_MPI_F08>)
+  set_target_properties(${modName} PROPERTIES Fortran_PREPROCESS "ON"
+  )# Use preprocessor on .f90 files
```

Comment:
I think some of the comments in CMake files have a bit wonky formatting now, but I don't mind much. Just pointing it out in case you didn't notice.

Reply:
Yes, some things are not great and had to fix a few thing manually already. I'll have another look, thank you for pointing these out!